build.gradle

```diff
@@ -1,5 +1,6 @@
 ext {
     wiremockVersion = '3.9.0'
+    wiremockStandaloneVersion = '3.0.1'
```

Comment:
Is there any reason why the standalone version needs to be pinned to this release?  Could it be the latest version ?

Reply:
No particular reason, I think I just missed this. I have completely removed this property now and used the `wiremockVersion` declared above. Is it ok - or do you prefer a different property just for wiremock-standalone?